UI/SKCanvasElementShowcase/ReadMe.md

```diff
@@ -0,0 +1,5 @@
+# SKCanvasElementShowcase
+
+![SKCanvasElementShowcase screenshot](./doc/assets/screenshot.png)
+
+This sample shows how to use `SKCanvasElement`. The sample contains a `Slider` that slides between different Skia drawings. For more information, [read our docs](https://aka.platform.uno/skcanvaselement).
```

Comment:
@jeromelaban This link will need to be valid (and maybe added to the announcement of the next release)

Reply:
It will be at the time of the release.